calibration/ADCPSL/CGINS-ADCPSL-24544__20171109.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+24544,CC_scale_factor1,0.45,constant; CP02PMUI-00010
+24544,CC_scale_factor2,0.45,constant
+24544,CC_scale_factor3,0.45,constant
+24544,CC_scale_factor4,0.45,constant
+24544,CC_bin_size,400.0,in cm
+24544,CC_dist_first_bin,901.0,in cm
+24544,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```

Comment:
shouldn't this be an ADCPTG?

Reply:
Yes it should.  My typo on the Spreadsheet used to generate these.